src/main/java/seedu/address/logic/commands/UnstarCommand.java

```diff
@@ -55,7 +55,8 @@ public CommandResult execute(Model model) throws CommandException {
         contactToUnstar.unstarContact();
         Person unstarredContact = new Person(contactToUnstar.getName(), contactToUnstar.getPhone(),
                 contactToUnstar.getEmail(), contactToUnstar.getAddress(), contactToUnstar.getCompany(),
-                contactToUnstar.getPriority(), contactToUnstar.isStarred(), contactToUnstar.getTags());
+                contactToUnstar.getMeeting(), contactToUnstar.getPriority(), contactToUnstar.isStarred(),
+                contactToUnstar.getTags());
```

Comment:
It's nice that you followed the java coding standard in leaving a line for a long statement